tests/integration/buildcmd/test_build_cmd_arm64.py

```diff
@@ -36,22 +36,28 @@ class TestBuildCommand_PythonFunctions_With_Specified_Architecture_arm64(BuildIn
             ("python3.9", "Python", False),
             ("python3.10", "Python", False),
             ("python3.11", "Python", False),
-            ("python3.12", "Python", False),
```

Comment:
Does the test also perform an invoke? If not, then we can leave the ones with `use_container=False` in the old AppVeyor jobs. Same comment for the other AL2023 based runtimes that had that container `use_container` check